### PR TITLE
tikv: do not capture output in tikv ghpr test

### PR DESCRIPTION
**Pull request by glorv**
add `--nocapture` to `cargo nextest run` for better panic debugging.

### CODE REVIEW

**Comment by glorv** [comment]
@purelind @Connor1996 PTAL

File: jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy [hunk 1]
```diff
@@ -506,7 +506,7 @@ stage('Test') {
                                 curl -o \$i ${FILE_SERVER_URL}/download/tikv_test/${ghprbActualCommit}/\$new_string --create-dirs;
                                 chmod +x \$i;
                             done
-                            if cargo nextest run -P ci --binaries-metadata test-binaries.json --cargo-metadata test-metadata.json --partition count:${chunk_suffix}/${CHUNK_COUNT} ${EXTRA_NEXTEST_ARGS}; then
+                            if cargo nextest run -P ci --binaries-metadata test-binaries.json --cargo-metadata test-metadata.json --partition count:${chunk_suffix}/${CHUNK_COUNT} --nocapture ${EXTRA_NEXTEST_ARGS}; then
```
Comment (Connor1996) [inline jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy:509]: Can we pass it by `EXTRA_NEXTEST_ARGS`?

Comment (glorv) [reply jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy:509]: Is there any different? I suppose `EXTRA_NEXTEST_ARGS` is used for extra arguments passed from PR comments.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by purelind** [comment]
wait for replay test result  https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/39951/pipeline

**Comment by purelind** [comment]
> wait for replay test result https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/39951/pipeline

test failed 
image

**Comment by glorv** [comment]
Found `--nocapture` is conflict with `--test-threads`, so I will close this PR. We may try to find other solution later.

**Pull request closed without merging**